app/package.json

```diff
@@ -9,6 +9,10 @@
     "lint": "next lint"
```

Comment:
I wonder if we can move these down to the componenet level -- making each one of our components an indepdendently installable npm module (optionally).

Reply:
I guess we can achieve this by making a monorepo and optionally dealing with components as we need. I will talk with Rohan sir about this and let you know